extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfo.java

```diff
@@ -157,31 +154,18 @@ private static void extractStreams(final StreamInfo streamInfo,
             streamInfo.addError(new ExtractionException("Couldn't get audio streams", e));
         }
 
-        /* Extract video stream url */
         try {
             streamInfo.setVideoStreams(extractor.getVideoStreams());
         } catch (final Exception e) {
             streamInfo.addError(new ExtractionException("Couldn't get video streams", e));
         }
 
-        /* Extract video only stream url */
         try {
             streamInfo.setVideoOnlyStreams(extractor.getVideoOnlyStreams());
         } catch (final Exception e) {
             streamInfo.addError(new ExtractionException("Couldn't get video only streams", e));
         }
 
-        // Lists can be null if an exception was thrown during extraction
-        if (streamInfo.getVideoStreams() == null) {
-            streamInfo.setVideoStreams(Collections.emptyList());
-        }
-        if (streamInfo.getVideoOnlyStreams() == null) {
-            streamInfo.setVideoOnlyStreams(Collections.emptyList());
-        }
-        if (streamInfo.getAudioStreams() == null) {
-            streamInfo.setAudioStreams(Collections.emptyList());
-        }
```

Comment:
Why are you removing this?

Reply:
Because the default values are now set in the corresponding attributes.